content/en/docs/rules/overriding.md

```diff
@@ -0,0 +1,273 @@
+---
+title: Overriding Rules
+description: Overriding Falco rules
+linktitle: Overriding Rules
+weight: 50
+---
+## Overview
+There may be cases where you need to adjust the behavior of the Falco-supplied {{< glossary_tooltip text="list" term_id="lists" >}}, {{< glossary_tooltip text="macro" term_id="macros" >}}, and  {{< glossary_tooltip text="rule" term_id="rules" >}}.
+
+To enable this Falco allows you to define multiple rules files. The additional rules files can be used to add new lists, macros and rules or to override (modify) existing ones.
```

Comment:
We could note that you can also override in the same file, as long as the override is specified after the definition.

Reply:
Done!